contributors/devel/scheduler.md

```diff
@@ -1,39 +1,63 @@
 # The Kubernetes Scheduler
 
-The Kubernetes scheduler runs as a process alongside the other master
-components such as the API server. Its interface to the API server is to watch
-for Pods with an empty PodSpec.NodeName, and for each Pod, it posts a Binding
-indicating where the Pod should be scheduled.
+The Kubernetes scheduler runs as a process alongside the other master components such as the API server.
+Its interface to the API server is to watch for Pods with an empty PodSpec.NodeName,
```

Comment:
@jayunit100 
I see no problem in this sentence:

"Its interface to the API server is to watch..."

Can you clarify what it should be?